backend/src/Service/ResetPasswordService.php

```
@@ -43,18 +44,37 @@ public function sendPasswordResetEmail(string $email): bool
+            ]);
+
             return true;
         }
-        
+
         // Invalidate any existing tokens for this user
         $this->tokenRepository->invalidateAllTokensForUser($user);
-        
+
         // Create a new token
         [$token, $plainToken] = $this->createToken($user);
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$token'. (undefined)

(UnusedLocalVariable)